src/Symfony/Component/Uid/Tests/Command/GenerateUlidCommandTest.php

```diff
@@ -89,4 +89,15 @@ public function testFormat()
 
         Ulid::fromRfc4122(trim($commandTester->getDisplay()));
     }
+
+    public function testTimestampIncrementWhenGeneratingSeveralTimeBasedUlids()
```

Comment:
testTimestampIncrementWhenGeneratingSeveral~TimeBased~Ulids

Reply:
fixed in 1b93740325